game.py

```diff
@@ -26,16 +34,25 @@ class AI(Bot):
     pass
 
 
+class Greedy(Bot):
+    """
+    Bot that always takes as many stones as possible
+    """
+    def move_strategy(self, state):
+        pile_index = [i for i, pile in enumerate(state) if pile == max(state)]
```

Comment:
```
In [1]: foo = [1,2,3,2,1]

In [2]: max(foo)
Out[2]: 3

In [3]: foo.index(3)
Out[3]: 2

In [4]: foo.index(max(foo))
Out[4]: 2
```

Reply:
Yessss, I knew there must be a better way.